x-pack/plugins/canvas/shareable_runtime/webpack.config.js

```diff
@@ -189,10 +189,7 @@ module.exports = {
         exclude: /node_modules/,
       },
       {
-        test: [
-          require.resolve('@elastic/eui/es/components/drag_and_drop'),
-          require.resolve('highlight.js'),
```

Comment:
Removing `react-syntax-highlighter` caused this to fail as a lint, as it was pulling in an extra dependency which was being excluded from a build process here for some reason. But since `react-syntax-highlighter` is no longer present, the test for null-loading `highlight.js` no longer needs to be there.